apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/notices.js

```diff
@@ -117,7 +127,12 @@ function GlobalStylesEditNotice() {
 	}, [ editEntityRecord, globalStylesId, isSiteEditor ] );
 
 	const openResetGlobalStylesSupport = useCallback( () => {
-		window.open( 'https://wordpress.com/support/using-styles/#reset-all-styles', '_blank' ).focus();
+		window
+			.open(
+				localizeUrl( 'https://wordpress.com/support/using-styles/#reset-all-styles' ),
```

Comment:
Does this actually work?
I think it needs something like this (untested):
```js
import { useLocalizeUrl } from '@automattic/i18n-utils';

const localizeUrl = useLocalizeUrl();
```

Reply:
Why wouldn't it work? Looks like `useLocalizeUrl` just provides some default args to `localizeUrl` but it looks like the defaults are all fine here?

Reply:
Ok, I tried it and it didn't work, I still got English. It looks to me that `localizeUrl` uses [`getLocaleSlug`](https://github.com/Automattic/wp-calypso//blob/style-variations-wording/newsletter/packages/i18n-calypso/src/i18n.js#L310) which uses the locale from the state though.

Reply:
Right, apologies for the code suggestion. I think I was looking at the wrong PR checked out and could not see the `localizeUrl` import. 🤦 

But as Dean confirmed, this is not working.
Or at least it didn't work for me with the account set on Italian, which has the [corresponding support page](https://wordpress.com/it/support/utilizzo-di-stili/), even though it's not as thorough, and it's missing the `reset-all-styles` section.

The English and Italian URLs are very different; it's not just a matter of locale slug.
Maybe `localizeUrl` does not work with support pages? 🤔 

* English: `https://wordpress.com/support/using-styles/`
* Italian: `https://wordpress.com/it/support/utilizzo-di-stili/`

Reply:
This works the same as before the change. The change is only there to please the linter so I guess we can move on with this (no regressions as it continues to work as before)